toxcore/group_chats.c

```diff
@@ -227,9 +231,13 @@ GC_Connection *get_gc_connection(const GC_Chat *chat, int peer_number)
 }
 
 /** Returns the amount of empty padding a packet of designated length should have. */
-static uint16_t group_packet_padding_length(uint16_t length)
+static uint16_t group_packet_padding_length(uint16_t length, uint8_t net_packet_type)
 {
-    return (MAX_GC_PACKET_CHUNK_SIZE - length) % GC_MAX_PACKET_PADDING;
+    if (net_packet_type == NET_PACKET_GC_LOSSY) {
```

Comment:
The if/else branches are almost identical except for 1 component of the expression. You could use ?: to shorten it and make it clearer that the branches are mostly the same.

Reply:
it became a function, this comment is now misaligned

Reply:
Thoughts on making it a single `?:` return rather than if/else return?

Reply:
Should it be a switch case then?

Reply:
hmmmm, but a switch feels excessive too. man, im kinda split on this.

Reply:
Let's leave it like this for now.